pkgs/tools/inputmethods/fcitx5/fcitx5-bamboo.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, fcitx5
+, gettext
+, go
+}:
+stdenv.mkDerivation rec {
+  pname = "fcitx5-bamboo";
+  version = "1.0.4";
+
+  src = fetchFromGitHub {
+    owner = "fcitx";
+    repo = "fcitx5-bamboo";
+    rev = version;
```

Comment:
```suggestion
    rev = finalAttrs.version;
```

Reply:
nah, I disagree, this is not worth it when you have to override the fetcher when overriding `version`.